src/contexts/collectionEditor/CollectionEditorContext.tsx

```diff
@@ -1,32 +1,38 @@
 import { createContext, memo, ReactNode, useCallback, useContext, useMemo, useState } from 'react';
 import { arrayMove } from '@dnd-kit/sortable';
-import { DragEndEvent } from '@dnd-kit/core';
-import { EditModeToken, StagingItem } from 'flows/shared/steps/OrganizeCollection/types';
-import { UpdateCollectionTokensInput } from '__generated__/useUpdateCollectionTokensMutation.graphql';
+import { DragEndEvent, UniqueIdentifier } from '@dnd-kit/core';
+import {
+  EditModeToken,
+  StagedCollection,
+  StagingItem,
+} from 'flows/shared/steps/OrganizeCollection/types';
+import { generate12DigitId } from 'utils/collectionLayout';
```

Comment:
supernit: why not use [uuid](https://www.npmjs.com/package/uuid)?

Reply:
oh yeah I think we had this convo before 😅 totally down for uuid, I just stuck with what we're using for the whitespace blocks. and since we don't save these ids to the db so I wasn't pressed to change it..  maybe we can swap it all out as a follow up or the next we visit the editor for multi chain!